src/scenic/simulators/carla/model.scenic

```diff
@@ -90,6 +97,8 @@ class Vehicle(Vehicle, CarlaActor, Steers):
     def setReverse(self, reverse):
         self.control.reverse = reverse
 
+    def _getClosestTrafficLight(self, distance=100):
```

Comment:
No, the underscore is explicitly there to convey that that function shouldn't be used directly from a scenario because it returns a CARLA actor. Regarding the distance argument, we are using CARLA's *get_landmarks_of_type(self, distance, type, stop_at_junction=False)*, which checks all the landmarks within that distance. Therefore, the distance cannot be infinite, or it will never stop searching.